script/local

```diff
@@ -7,13 +7,21 @@ set -e -u
 cd "$(dirname "$0")/.."
 
 # Parse command line arguments.
-if [ $# -ne 1 ]; then
-	echo "usage: $0 envfile" >&2
+if [ $# -lt 1 ]; then
+	echo "usage: $0 envfile [notebooks_path]" >&2
 	exit 2
 fi
+
 export ENV_FILE=$1
 echo "Env file: ${ENV_FILE}"
 
+export NOTEBOOKS="./notebooks"
+
+if [ $# -ge 2 ]; then
+	export NOTEBOOKS=$2
+fi
+echo "Notebooks dir: ${NOTEBOOKS}"
+
```

Comment:
I would do something like this:

```suggestion
ENV_FILE=""
NOTEBOOKS="./notebooks"
case $# in
    1)
        ENV_FILE=$1
        ;;
    2)
        ENV_FILE=$1
        NOTEBOOKS=$2
        ;;
    *)
        echo "usage: $0 envfile [notebooks_path]" >&2
        exit 2
esac

export ENV_FILE
export NOTEBOOKS
```

Watch out! I've not tested it.

Reply:
Yes it's cleaner than the current solution, changing and testing it.